docs/fundamentals/code-analysis/quality-rules/ca1062.md

```diff
@@ -48,13 +48,16 @@ You can suppress a warning from this rule if you are sure that the dereferenced
 
 Use the following options to configure which parts of your codebase to run this rule on.
 
+- [Include specific API surfaces](#include-specific-api-surfaces)
 - [Exclude specific symbols](#exclude-specific-symbols)
 - [Exclude specific types and their derived types](#exclude-specific-types-and-their-derived-types)
 - [Exclude extension method 'this' parameter](#exclude-extension-method-this-parameter)
 - [Null check validation methods](#null-check-validation-methods)
 
 These options can be configured for just this rule, for all rules, or for all rules in this category ([Design](design-warnings.md)). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
 
+[!INCLUDE[api-surface](~/includes/code-analysis/api-surface.md)]
```

Comment:
Do we need to specify that the option is supported only in .NET7 and later?

Reply:
How do we handle such change generally? Like every new user option is only available on new version of the code, right? For example, do we have the same concern with #28234?

Reply:
We should note the version somehow. We should also add CA1062 to this list: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-quality-rule-options#api_surface.

```suggestion
[!INCLUDE[api-surface](~/includes/code-analysis/api-surface.md)]

> [!NOTE]
> This option is supported for CA1062 in .NET 7 and later versions only.
```